test/examples/make.jl

```diff
@@ -213,7 +215,26 @@ else
     nothing
 end
 
-# same as HTML, but with MathJax3
+# Same as HTML, but with variations on the MathJax configuration
+examples_html_mathjax2_custom_doc = if "html-mathjax2-custom" in EXAMPLE_BUILDS
+    @info("Building mock package docs: HTMLWriter / deployment build using MathJax v2 (custom URL)")
+    html_doc("html-mathjax2-custom",
+        MathJax2(
+            Dict(
+                :loader => Dict("load" => ["[tex]/physics"]),
+                :tex => Dict(
+                    "inlineMath" => [["\$","\$"], ["\\(","\\)"]],
+                    "tags" => "ams",
+                    "packages" => ["base", "ams", "autoload", "physics"],
+                ),
+            ),
+            url = "https://cdn.jsdelivr.net/npm/mathjax@2/MathJax.js",
+        ),
+    )
+else
+    @info "Skipping build: HTML/deploy MathJax v2 (custom URL)" EXAMPLE_BUILDS get(ENV, "DOCUMENTER_TEST_EXAMPLES", nothing)
+    nothing
+end
```

Comment:
This build actually doesn't completely work (MathJax loads, but does not render the equation at the end of `man/tutorials.md`), but I am not sure why at the moment.

Reply:
I don't have so much knowledge of MathJax2/3, but the config part in the example of MathJax2 above is the same as the one of MathJax3.
```
MathJax3(
    Dict(
        :loader => Dict("load" => ["[tex]/physics"]),
        :tex => Dict(
            "inlineMath" => [["\$","\$"], ["\\(","\\)"]],
            "tags" => "ams",
            "packages" => ["base", "ams", "autoload", "physics"],
        ),
    ),
 ),
```